scripts/cybershake/install_cybershake_fault.py

```diff
@@ -54,72 +62,66 @@ def main():
     version = cybershake_cfg["version"]
     v1d_full_path = cybershake_cfg["v_1d_mod"]
 
-    site_v1d_dir = None
-    hf_stat_vs_ref = None
-    if cybershake_cfg.get("site_v1d_dir") is not None:
-        site_v1d_dir = cybershake_cfg["site_v1d_dir"]
-    if cybershake_cfg.get("hf_stat_vs_ref") is not None:
-        hf_stat_vs_ref = cybershake_cfg["hf_stat_vs_ref"]
+    site_v1d_dir = cybershake_cfg.get("site_v1d_dir")
+    hf_stat_vs_ref = cybershake_cfg.get("hf_stat_vs_ref")
 
-    # this variable seems to not be used anywhere important.
-    run_dir = sim_root_dir
-    user_root = path_cybershake
     stat_file_path = cybershake_cfg['stat_file_path']
     vs30_file_path = stat_file_path.replace('.ll', '.vs30')
     vs30ref_file_path = stat_file_path.replace('.ll', '.vs30ref')
 
     error_log = os.path.join(path_cybershake, "install_error.log")
     params_vel = "params_vel.py"
 
-    source = args.vm
+    fault_name = args.vm
 
     # this variable has to be empty
     # TODO: fix this legacy issue, very low priority
     event_name = ""
 
-    run_name = source
-    vel_mod_dir = os.path.join(vm_root_dir, source)
+    # Get & validate velocity model directory
+    vel_mod_dir = os.path.join(vm_root_dir, fault_name)
     valid_vm, message = validate_vm.validate_vm(vel_mod_dir)
     if not valid_vm:
-        message = "Error: VM {} failed {}".format(source, message)
+        message = "Error: VM {} failed {}".format(fault_name, message)
         print(message)
         with open(error_log, "a") as error_fp:
             error_fp.write(message)
         exit()
 
+    # Load the variables from params_vel.py
     params_vel_path = os.path.join(vel_mod_dir, params_vel)
```

Comment:
So we end up with a params vel.py, params_vel.json and vm_params.yaml?

Reply:
I believe this is the current state of things. I agree its seem a bit over the top haha.